image/templates/helm/stackrox-central/templates/01-central-13-deployment.yaml.htpl

```diff
@@ -142,6 +142,10 @@ spec:
           value: "true"
         {{- end }}
         [<- end >]
+        [< if .FeatureFlags.ROX_POLICY_AS_CODE >]
```

Comment:
Please use `[<- ... >]` for this kind of line-based control-flow. Reasons are explaines in https://github.com/stackrox/stackrox/blob/fd6f4c86f36f2b4d6ebd3a56ba4bdb35fbabf177/image/templates/CHANGING_CHARTS.md#whitespace-elimination and apply equally to the meta-templating.